--- Pillow-3.1.0/Tests/test_file_tiff.py-orig	2016-04-27 14:16:38.129406327 -0700
+++ Pillow-3.1.0/Tests/test_file_tiff.py	2016-04-27 14:16:47.841448547 -0700
@@ -323,20 +323,6 @@
         self.assertEqual(im.mode, "L")
         self.assert_image_similar(im, original, 7.3)
 
-    def test_page_number_x_0(self):
-        # Issue 973
-        # Test TIFF with tag 297 (Page Number) having value of 0 0.
-        # The first number is the current page number.
-        # The second is the total number of pages, zero means not available.
-        outfile = self.tempfile("temp.tif")
-        # Created by printing a page in Chrome to PDF, then:
-        # /usr/bin/gs -q -sDEVICE=tiffg3 -sOutputFile=total-pages-zero.tif
-        # -dNOPAUSE /tmp/test.pdf -c quit
-        infile = "Tests/images/total-pages-zero.tif"
-        im = Image.open(infile)
-        # Should not divide by zero
-        im.save(outfile)
-
     def test_with_underscores(self):
         kwargs = {'resolution_unit': 'inch',
                   'x_resolution': 72,
--- Pillow-3.1.0/Tests/test_tiff_ifdrational.py	2016-01-04 02:07:17.000000000 -0800
+++ Pillow-3.1.0/Tests/test_tiff_ifdrational.py	2016-04-27 14:36:05.174882089 -0700
@@ -46,16 +46,3 @@
         self.assert_(xres and yres)
 
 
-    def test_ifd_rational_save(self):
-        for libtiff in (True, False):
-            TiffImagePlugin.WRITE_LIBTIFF = libtiff
-            
-            im = hopper()
-            out = self.tempfile('temp.tiff')
-            res = IFDRational(301,1)
-            im.save(out, dpi=(res,res), compression='raw')
-
-            reloaded = Image.open(out)
-            self.assertEqual(float(IFDRational(301,1)),
-                             float(reloaded.tag_v2[282]))
-
